src/concrete/ml/torch/hybrid_model.py

```diff
@@ -1,16 +1,44 @@
 """Implement the conversion of a torch model to a hybrid fhe/torch inference."""
 
+import ast
+import enum
+import io
+import sys
+import time
 import uuid
 from pathlib import Path
-from typing import List, Optional, Union
+from typing import Dict, List, Optional, Tuple, Union
 
+import numpy
+import requests
 import torch
 from concrete.fhe import Configuration
 from torch import nn
 from transformers import Conv1D
 
 from ..deployment.fhe_client_server import FHEModelClient, FHEModelDev
-from .compile import compile_torch_model
+from .compile import QuantizedModule, compile_torch_model
+
+
+class FHEMode(enum.Enum):
```

Comment:
hmm, I don't think it's a good idea to have another `FHEMode` instance here while we already have a similar (but different) one in  `common/utils.py`. Could you either inherit from that latter or just name it differently ?

Reply:
Indeed I'll rename